### PR TITLE
Saul 61 AND 62

### PR DESCRIPTION
**Pull request by Renkon**
show and delete all in one, sorry about branch name

### CODE REVIEW

**Review by NahuelBerg — APPROVED** [review]

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_streaming_endpoint_scenarios.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            'description': 'test streaming description',
+            'maxCacheAge': 11,
+            'scaleUnits': 4,
+            'tags': 'foo=bar',
+            'clientAccessPolicy': self._get_test_data_file('clientAccessPolicy.xml'),
+            'crossDomainPolicy': self._get_test_data_file('crossDomainPolicy.xml')
+        })
+
+        self.cmd('az ams account create -n {amsname} -g {rg} --storage-account {storageAccount} -l {location}', checks=[
+            self.check('name', '{amsname}'),
```
Comment (damatojuan) [inline src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_streaming_endpoint_scenarios.py]: It's not necessary to add checks here

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_streaming_endpoint_scenarios.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            self.check('location', 'West US 2')
+        ])
+
+        self.cmd('az ams streaming endpoint create -g {rg} -a {amsname} -n {streamingEndpointName} -l {location} --availability-set-name {availabilitySetName} --cdn-provider {cdnProvider} --cdn-profile {cdnProfile} --description "{description}" --max-cache-age {maxCacheAge} --scale-units {scaleUnits} --tags "{tags}" --client-access-policy "{clientAccessPolicy}" --cross-domain-policy "{crossDomainPolicy}"')
+
+        self.cmd('az ams streaming endpoint show -g {rg} -a {amsname} --streaming-endpoint-name {streamingEndpointName}', checks=[
```
Comment (damatojuan) [inline src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_streaming_endpoint_scenarios.py]: streaming-endpoint-name shouldn't be the parameter name. For every show operation we are using --name/-n. Check other commands.

**Review by damatojuan — CHANGES_REQUESTED** [review]

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_streaming_endpoint_scenarios.py (hunk 1 above)
Comment (Renkon) [reply src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_streaming_endpoint_scenarios.py]: Fixed in https://github.com/southworkscom/azure-cli/pull/28/commits/30f2c5023ec3d2c7d2fa2d7ac2a7d1ed981bb16e

File: src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_streaming_endpoint_scenarios.py (hunk 2 above)
Comment (Renkon) [reply src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_streaming_endpoint_scenarios.py]: done

**Review by damatojuan — APPROVED** [review]

**Pull request merged**